src/components/current-query/index.js

```diff
@@ -59,8 +69,9 @@ class CurrentQuery extends React.Component {
   render() {
     const {bootstrapCss, query} = this.props;
 
+
     const splitFields = query.searchFields
-      .filter((searchField) => searchField.value && searchField.value.length > 0)
+      .filter((searchField) => searchField.value && (searchField.value.length > 0 || Object.keys(searchField.value).length > 0))
```

Comment:
The value of spatial query type is the dictionary. We need to check if the dictionary has value.